sdk/search/azure-search/out/azure/search/index/_index_batch.pyi

```diff
@@ -0,0 +1,13 @@
+from ._generated.models import IndexAction as IndexAction
+from typing import Any, List
+
+def flatten_args(args: Any): ...
+
+class IndexBatch:
+    def __init__(self) -> None: ...
+    def add_upload_documents(self, *documents: Any) -> None: ...
```

Comment:
Do we support calling the method with both:

```python
# This works...
index_batch.add_upload_documents([doc1, doc2])
# As does this...
index_batch.add_upload_documents(*[doc1, doc2])
```

or is only the latter version supported?

Reply:
I thought we had resolved to afford both :) But also happy to change to a single list arg only

Reply:
Yep, so did I. But the API listing does not show that, so I just wanted to make sure!